3-extra/2-sorting-algorithm.js

```diff
@@ -2,20 +2,54 @@
 At the start of the course, you worked in teams to sort your team members, labelled by
 numbers, in ascending or descending order.
 
-Today, you will be applying the sorting algorithm you used in that exercise in code! 
+Today, you will be applying the sorting algorithm you used in that exercise in code!
 
 Create a function called sortAges which:
 - takes an array of mixed data types as input
 - removes any non-number data types without using the built-in javascript filter method
-- returns an array of sorted ages in ascending order 
+- returns an array of sorted ages in ascending order
   - HARD MODE - without using the built-in javascript sort method 😎
 
 You don't have to worry about making this algorithm work fast! The idea is to get you to
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {}
+function sortAges(arr) {
+  let num = [];
+  arr.forEach((element) => {
+    typeof element === "number" ? num.push(element) : null;
+  });
 
+  for (let i = 0; i < num.length; i++) {
+    for (let index = i + 1; index < num.length; index++) {
+      if (num[i] > num[index]) {
+        let templet = num[i];
+        num[i] = num[index];
+        num[index] = templet;
+      }
+    }
+  }
+  return num;
```

Comment:
Good, this work well. There are a few different ways of doing this - if you have time, ask other students how they have done this.